src/pytest_plugins/consume/direct/conftest.py

```diff
@@ -7,34 +7,52 @@
 
 import json
 import tempfile
+import warnings
 from pathlib import Path
 from typing import List
 
 import pytest
 
+from ethereum_clis.ethereum_cli import EthereumCLI
 from ethereum_clis.fixture_consumer_tool import FixtureConsumerTool
 from ethereum_test_base_types import to_json
+from ethereum_test_fixtures import BaseFixture
 from ethereum_test_fixtures.consume import TestCaseIndexFile, TestCaseStream
 from ethereum_test_fixtures.file import Fixtures
 
 from ..consume import FixturesSource
 
 
+class CollectOnlyCLI(EthereumCLI):
+    """A dummy CLI for use with `--collect-only`."""
+
+    def __init__(self):  # noqa: D107
+        pass
+
+
+class CollectOnlyFixtureConsumer(
+    FixtureConsumerTool, CollectOnlyCLI, fixture_formats=list(BaseFixture.formats.values())
+):
+    """A dummy fixture consumer for use with `--collect-only`."""
+
+    def consume_fixture(self):  # noqa: D102
+        pass
+
+
 def pytest_addoption(parser):  # noqa: D103
     consume_group = parser.getgroup(
         "consume_direct", "Arguments related to consuming fixtures via a client"
     )
 
     consume_group.addoption(
-        "--fixture-consumer-bin",
+        "--bin",
         action="append",
         dest="fixture_consumer_bin",
-        type=list,
-        default=[Path("evm")],
+        type=str,
```

Comment:
```suggestion
        type=Path,
```
Works too, I just checked.

Reply:
Oh yeah, that's perfect, thanks.